tools/check_repositories.sh

```diff
@@ -12,7 +12,12 @@ fi
 
 # Check whether number of defined `url =` or `urls =` and `sha256 =` kwargs in
 # repository definitions is equal.
-urls_count=$(grep -IPnrs "url(s)? =" --include=*.bzl . | wc -l)
+if [[ `uname` == "Darwin" ]]; then
+  # When running on OSX use egrep instead of perl regexp
```

Comment:
Can't we just use `egrep` on both?

Reply:
👍 I can update it to do that but it'll change from `perl-regexp` to `extended-regexp` for linux. Probably won't cause any issues though.